vision_agent_tools/tools/clip_media_sim.py

```diff
@@ -54,13 +71,24 @@ def __call__(
         frame_embs = []
         for frame in video:
             inputs = self.processor(images=frame, return_tensors="pt")
-            with torch.no_grad(), torch.autocast(self.device):
+            with torch.autocast(self.device):
                 inputs.to(self.device)
                 outputs = self.model.get_image_features(**inputs)
-            frame_embs.append(outputs.detach())
+            frame_embs.append(outputs.squeeze().detach())
         frame_embs = torch.stack(frame_embs)
-        sims = F.cosine_similarity(frame_embs, target, dim=-1).cpu().numpy()
-        times = np.array(timestamps)
-        output = np.concatenate([times[:, None], sims], axis=1)
+
+        sims = (
```

Comment:
I forgot this in the file I sent, but these would be the logits right? so we need to still apply softmax